src/main/java/com/here/platform/artifact/maven/wagon/ArtifactWagon.java

```diff
@@ -123,7 +123,7 @@ public class ArtifactWagon extends AbstractHttpClientWagon {
 
   private final Object lock = new Object();
   private final ObjectMapper objectMapper;
-  private String defaultArtifactServiceUrl;
+  private static String defaultArtifactServiceUrl;
```

Comment:
Please move variable up to the other static variable. Also it wold be nice to add comment why we made this variable static